python/astro_metadata_translator/tests.py

```diff
@@ -174,15 +180,15 @@ def assertObservationInfo(self, header, check_wcs=True, wcs_params=None, **kwarg
         # Check the properties
         for property, expected in kwargs.items():
             calculated = getattr(obsinfo, property)
-            msg = f"Comparing property {property}"
-            if isinstance(expected, u.Quantity):
+            msg = f"Comparing property {property} using translator {translator}"
+            if isinstance(expected, u.Quantity) and calculated is not None:
```

Comment:
Inside this if statement we extract the value of `calculated`. It can be `None` and so this check is required (it was added when some tests were blowing up). The `expected` value is a Quantity.

Reply:
Oh, the property can have a value of `None`.  I was missing that.  I thought you were expecting `getattr` to return `None` if the property didn't exist, but it should throw an exception in that case.

Reply:
Throwing an exception on failure to translate is optional for the `ObservationInfo` translator.

Reply:
Sorry.  I just meant that `getattr` throws a `NameError` if the attribute does not exist.